Sources/MessagingInApp/autogenerated/AutoLenses.generated.swift

```diff
@@ -46,8 +46,10 @@ struct Lens<Whole, Part> {
 }
 
 func * <A, B, C>(lhs: Lens<A, B>, rhs: Lens<B, C>) -> Lens<A, C> {
-    Lens<A, C>(get: { a in rhs.get(lhs.get(a)) },
-               set: { c, a in lhs.set(rhs.set(c, lhs.get(a)), a) })
+    Lens<A, C>(
+        get: { a in rhs.get(lhs.get(a)) },
+        set: { c, a in lhs.set(rhs.set(c, lhs.get(a)), a) }
```

Comment:
Do we care about the code coverage warning? In this case it's just being triggered by style changes, but I guess my question is, should this be covered? if so, when might we come back to this? Are we or should we track places to cover with tests as part of a clean-up activity?

Reply:
This particular file that you are talking about is generated for us. We tend to ignore those files entirely in code review because it's automatically generated. So, test coverage on this code isn't super important, IMO.